sopel/plugins/handlers.py

```diff
@@ -128,47 +130,47 @@ def is_loaded(self):
         raise NotImplementedError
 
     def setup(self, bot):
-        """Setup the plugin with the ``bot``
+        """Run the plugin's setup routine.
```

Comment:
If the subclass overrides are going to call it a "setup action", then this should too. I don't care _that_ much one way or the other, but I _do_ care about being consistent.

```suggestion
        """Run the plugin's setup action.
```